doc/content/ttn-lw-cli/ttn-lw-cli_applications_pubsubs_get.md

```diff
@@ -71,6 +71,8 @@ ttn-lw-cli applications pubsubs get [application-id] [pubsub-id] [flags]
       --service-data.topic                               select the service_data.topic field
       --uplink-message                                   select the uplink_message field and all allowed sub-fields
       --uplink-message.topic                             select the uplink_message.topic field
+      --uplink-normalized                                select the uplink_normalized field and all allowed sub-fields
```

Comment:
These changes are part of the current `v3.21`, but they did not make it into the `v3.21.1` release - are you sure you checked out the tag ?

Reply:
@adriansmares can you please check again? I see some messages are deleted, not sure if that's intended or I messed up the PR somehow.